independent-publisher-connectors/Sessionize/apiDefinition.swagger.json

```diff
@@ -0,0 +1,262 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Sessionize",
+    "description": "Sessionize is the smart way to manage Call for Papers, Speakers and Agenda for your conference.",
+    "version": "1.0",
+    "contact": {
+      "name": "Nanddeep Nachan, Smita Nachan",
+      "url": "https://www.nanddeepnachanblogs.com/",
+      "email": "NanddeepNachan@gmail.com"
+    }
+  },
+  "host": "sessionize.com",
+  "basePath": "/api/v2/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/{ID}/view/Sessions": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "groupId": {
+                    "type": "string",
+                    "description": "groupId"
+                  },
+                  "groupName": {
+                    "type": "string",
+                    "description": "groupName"
+                  },
+                  "sessions": {
+                    "type": "array",
+                    "items": {
+                      "type": "object",
+                      "properties": {
+                        "questionAnswers": {
+                          "type": "array",
+                          "items": {},
+                          "description": "questionAnswers"
+                        },
+                        "id": {
+                          "type": "string",
+                          "description": "id"
+                        },
+                        "title": {
+                          "type": "string",
+                          "description": "title"
+                        },
+                        "description": {
+                          "type": "string",
+                          "description": "description"
+                        },
+                        "startsAt": {
+                          "type": "string",
+                          "description": "startsAt"
+                        },
+                        "endsAt": {
+                          "type": "string",
+                          "description": "endsAt"
+                        },
+                        "isServiceSession": {
+                          "type": "boolean",
+                          "description": "isServiceSession"
```

Comment:
@nanddeepn All the description values of properties in the swagger are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)